executor/scripts/test_local_executor.py

```
@@ -0,0 +1,256 @@
+                    f"  ✓ Backend is running: {data.get('name')} v{data.get('version')}"
+                )
+                return True
+            else:
+                print(f"  ✗ Backend returned status {response.status_code}")
+                return False
+        except httpx.ConnectError:
+            print(f"  ✗ Cannot connect to backend at {backend_url}")
+            print("    Please start the backend server first:")
+            print("    cd backend && uv run uvicorn app.main:app --port 8000")
+            return False
+        except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)